src/views/listview/list-view.html

```diff
@@ -24,7 +24,7 @@
 
         <div class="list-view-pf-actions"
              ng-if="(actionButtons && actionButtons.length > 0) || (menuActions && menuActions.length > 0)">
-          <button class="btn btn-default {{actionButton.class}}" ng-repeat="actionButton in actionButtons"
+          <button class="btn {{actionButton.class = actionButton.class || 'btn-default'}}" ng-repeat="actionButton in actionButtons"
```

Comment:
I don't think we should assign actionButton.class here. Just using {{actionButton.class || 'btn-default'}} should be sufficient.